{"uid":"4fa5d56184fe8ea8","name":"Card Creation","fullName":"tests.regression_ui.test_ui_regression.TestUIRegression#test_card_creation","historyId":"37b04428e9c82a14933f1043cf15ed79","time":{"start":1735261547475,"stop":1735261585296,"duration":37821},"status":"broken","statusMessage":"TypeError: not all arguments converted during string formatting","statusTrace":"self = <tests.regression_ui.test_ui_regression.TestUIRegression object at 0x7ffb96c5a210>\ndriver = <selenium.webdriver.chrome.webdriver.WebDriver (session=\"cd219cf8d8d160acc351929ada33a9c6\")>\n\n    @pytest.mark.TC000\n    @pytest.mark.TC004\n    @allure.title('Card Creation')\n    def test_card_creation(self, driver):\n        time.sleep(10)\n        with allure.step('Log in to Trello account.'):\n            self.test_login(driver)\n        with allure.step('Open a board with an existing list and create a new card'):\n            try:\n>               WebDriverWait(driver, 10).until(EC.element_to_be_clickable((By.XPATH, self.SCC.Board.BOARD_TITLE))).click()\n\ntests/regression_ui/test_ui_regression.py:146: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <selenium.webdriver.support.wait.WebDriverWait (session=\"cd219cf8d8d160acc351929ada33a9c6\")>\nmethod = <function element_to_be_clickable.<locals>._predicate at 0x7ffb96bf6980>\nmessage = ''\n\n    def until(self, method: Callable[[D], Union[Literal[False], T]], message: str = \"\") -> T:\n        \"\"\"Calls the method provided with the driver as an argument until the \\\n        return value does not evaluate to ``False``.\n    \n        :param method: callable(WebDriver)\n        :param message: optional message for :exc:`TimeoutException`\n        :returns: the result of the last call to `method`\n        :raises: :exc:`selenium.common.exceptions.TimeoutException` if timeout occurs\n        \"\"\"\n        screen = None\n        stacktrace = None\n    \n        end_time = time.monotonic() + self._timeout\n        while True:\n            try:\n                value = method(self._driver)\n                if value:\n                    return value\n            except self._ignored_exceptions as exc:\n                screen = getattr(exc, \"screen\", None)\n                stacktrace = getattr(exc, \"stacktrace\", None)\n            if time.monotonic() > end_time:\n                break\n            time.sleep(self._poll)\n>       raise TimeoutException(message, screen, stacktrace)\nE       selenium.common.exceptions.TimeoutException: Message: \nE       Stacktrace:\nE       #0 0x55587747f31a <unknown>\nE       #1 0x555876f956e0 <unknown>\nE       #2 0x555876fe43e6 <unknown>\nE       #3 0x555876fe4681 <unknown>\nE       #4 0x555877029b04 <unknown>\nE       #5 0x55587700848d <unknown>\nE       #6 0x555877026ed7 <unknown>\nE       #7 0x555877008203 <unknown>\nE       #8 0x555876fd6cc0 <unknown>\nE       #9 0x555876fd7c9e <unknown>\nE       #10 0x55587744cd0b <unknown>\nE       #11 0x555877450c92 <unknown>\nE       #12 0x555877439b3c <unknown>\nE       #13 0x555877451807 <unknown>\nE       #14 0x55587741f0df <unknown>\nE       #15 0x55587746e578 <unknown>\nE       #16 0x55587746e740 <unknown>\nE       #17 0x55587747e196 <unknown>\nE       #18 0x7f676429ca94 <unknown>\nE       #19 0x7f6764329c3c <unknown>\n\n/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/site-packages/selenium/webdriver/support/wait.py:105: TimeoutException\n\nDuring handling of the above exception, another exception occurred:\n\nself = <tests.regression_ui.test_ui_regression.TestUIRegression object at 0x7ffb96c5a210>\ndriver = <selenium.webdriver.chrome.webdriver.WebDriver (session=\"cd219cf8d8d160acc351929ada33a9c6\")>\n\n    @pytest.mark.TC000\n    @pytest.mark.TC004\n    @allure.title('Card Creation')\n    def test_card_creation(self, driver):\n        time.sleep(10)\n        with allure.step('Log in to Trello account.'):\n            self.test_login(driver)\n        with allure.step('Open a board with an existing list and create a new card'):\n            try:\n                WebDriverWait(driver, 10).until(EC.element_to_be_clickable((By.XPATH, self.SCC.Board.BOARD_TITLE))).click()\n                WebDriverWait(driver, 10).until(EC.visibility_of_element_located((By.XPATH, self.SCC.Board.LIST_TITLE)))\n                driver.find_element(By.XPATH, self.SCC.List.ADD_A_CARD_BUTTON).click()\n                driver.find_element(By.XPATH, self.SCC.List.CARD_NAME_FIELD).click()\n                time.sleep(1)\n                driver.find_element(By.XPATH, self.SCC.List.CARD_NAME_FIELD).send_keys('new_card')\n                driver.find_element(By.XPATH, self.SCC.List.ADD_CARD_SUBMIT_BUTTOMN).click()\n            except Exception as e:\n>               log.info(\"An error occurred: \", str(e))\n\ntests/regression_ui/test_ui_regression.py:154: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/logging/__init__.py:2216: in info\n    root.info(msg, *args, **kwargs)\n/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/logging/__init__.py:1539: in info\n    self._log(INFO, msg, args, **kwargs)\n/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/logging/__init__.py:1684: in _log\n    self.handle(record)\n/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/logging/__init__.py:1700: in handle\n    self.callHandlers(record)\n/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/logging/__init__.py:1762: in callHandlers\n    hdlr.handle(record)\n/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/logging/__init__.py:1028: in handle\n    self.emit(record)\n/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/site-packages/_pytest/logging.py:386: in emit\n    super().emit(record)\n/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/logging/__init__.py:1168: in emit\n    self.handleError(record)\n/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/logging/__init__.py:1160: in emit\n    msg = self.format(record)\n/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/logging/__init__.py:999: in format\n    return fmt.format(record)\n/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/site-packages/_pytest/logging.py:139: in format\n    return super().format(record)\n/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/logging/__init__.py:703: in format\n    record.message = record.getMessage()\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <LogRecord: root, 20, /home/runner/work/pythonSelenium/pythonSelenium/tests/regression_ui/test_ui_regression.py, 154, \"An error occurred: \">\n\n    def getMessage(self):\n        \"\"\"\n        Return the message for this LogRecord.\n    \n        Return the message for this LogRecord after merging any user-supplied\n        arguments with the message.\n        \"\"\"\n        msg = str(self.msg)\n        if self.args:\n>           msg = msg % self.args\nE           TypeError: not all arguments converted during string formatting\n\n/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/logging/__init__.py:392: TypeError","flaky":false,"newFailed":false,"newBroken":true,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[{"name":"driver","time":{"start":1735261547225,"stop":1735261547475,"duration":250},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"setup","time":{"start":1735261547475,"stop":1735261547475,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"testStage":{"status":"broken","statusMessage":"TypeError: not all arguments converted during string formatting","statusTrace":"self = <tests.regression_ui.test_ui_regression.TestUIRegression object at 0x7ffb96c5a210>\ndriver = <selenium.webdriver.chrome.webdriver.WebDriver (session=\"cd219cf8d8d160acc351929ada33a9c6\")>\n\n    @pytest.mark.TC000\n    @pytest.mark.TC004\n    @allure.title('Card Creation')\n    def test_card_creation(self, driver):\n        time.sleep(10)\n        with allure.step('Log in to Trello account.'):\n            self.test_login(driver)\n        with allure.step('Open a board with an existing list and create a new card'):\n            try:\n>               WebDriverWait(driver, 10).until(EC.element_to_be_clickable((By.XPATH, self.SCC.Board.BOARD_TITLE))).click()\n\ntests/regression_ui/test_ui_regression.py:146: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <selenium.webdriver.support.wait.WebDriverWait (session=\"cd219cf8d8d160acc351929ada33a9c6\")>\nmethod = <function element_to_be_clickable.<locals>._predicate at 0x7ffb96bf6980>\nmessage = ''\n\n    def until(self, method: Callable[[D], Union[Literal[False], T]], message: str = \"\") -> T:\n        \"\"\"Calls the method provided with the driver as an argument until the \\\n        return value does not evaluate to ``False``.\n    \n        :param method: callable(WebDriver)\n        :param message: optional message for :exc:`TimeoutException`\n        :returns: the result of the last call to `method`\n        :raises: :exc:`selenium.common.exceptions.TimeoutException` if timeout occurs\n        \"\"\"\n        screen = None\n        stacktrace = None\n    \n        end_time = time.monotonic() + self._timeout\n        while True:\n            try:\n                value = method(self._driver)\n                if value:\n                    return value\n            except self._ignored_exceptions as exc:\n                screen = getattr(exc, \"screen\", None)\n                stacktrace = getattr(exc, \"stacktrace\", None)\n            if time.monotonic() > end_time:\n                break\n            time.sleep(self._poll)\n>       raise TimeoutException(message, screen, stacktrace)\nE       selenium.common.exceptions.TimeoutException: Message: \nE       Stacktrace:\nE       #0 0x55587747f31a <unknown>\nE       #1 0x555876f956e0 <unknown>\nE       #2 0x555876fe43e6 <unknown>\nE       #3 0x555876fe4681 <unknown>\nE       #4 0x555877029b04 <unknown>\nE       #5 0x55587700848d <unknown>\nE       #6 0x555877026ed7 <unknown>\nE       #7 0x555877008203 <unknown>\nE       #8 0x555876fd6cc0 <unknown>\nE       #9 0x555876fd7c9e <unknown>\nE       #10 0x55587744cd0b <unknown>\nE       #11 0x555877450c92 <unknown>\nE       #12 0x555877439b3c <unknown>\nE       #13 0x555877451807 <unknown>\nE       #14 0x55587741f0df <unknown>\nE       #15 0x55587746e578 <unknown>\nE       #16 0x55587746e740 <unknown>\nE       #17 0x55587747e196 <unknown>\nE       #18 0x7f676429ca94 <unknown>\nE       #19 0x7f6764329c3c <unknown>\n\n/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/site-packages/selenium/webdriver/support/wait.py:105: TimeoutException\n\nDuring handling of the above exception, another exception occurred:\n\nself = <tests.regression_ui.test_ui_regression.TestUIRegression object at 0x7ffb96c5a210>\ndriver = <selenium.webdriver.chrome.webdriver.WebDriver (session=\"cd219cf8d8d160acc351929ada33a9c6\")>\n\n    @pytest.mark.TC000\n    @pytest.mark.TC004\n    @allure.title('Card Creation')\n    def test_card_creation(self, driver):\n        time.sleep(10)\n        with allure.step('Log in to Trello account.'):\n            self.test_login(driver)\n        with allure.step('Open a board with an existing list and create a new card'):\n            try:\n                WebDriverWait(driver, 10).until(EC.element_to_be_clickable((By.XPATH, self.SCC.Board.BOARD_TITLE))).click()\n                WebDriverWait(driver, 10).until(EC.visibility_of_element_located((By.XPATH, self.SCC.Board.LIST_TITLE)))\n                driver.find_element(By.XPATH, self.SCC.List.ADD_A_CARD_BUTTON).click()\n                driver.find_element(By.XPATH, self.SCC.List.CARD_NAME_FIELD).click()\n                time.sleep(1)\n                driver.find_element(By.XPATH, self.SCC.List.CARD_NAME_FIELD).send_keys('new_card')\n                driver.find_element(By.XPATH, self.SCC.List.ADD_CARD_SUBMIT_BUTTOMN).click()\n            except Exception as e:\n>               log.info(\"An error occurred: \", str(e))\n\ntests/regression_ui/test_ui_regression.py:154: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/logging/__init__.py:2216: in info\n    root.info(msg, *args, **kwargs)\n/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/logging/__init__.py:1539: in info\n    self._log(INFO, msg, args, **kwargs)\n/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/logging/__init__.py:1684: in _log\n    self.handle(record)\n/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/logging/__init__.py:1700: in handle\n    self.callHandlers(record)\n/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/logging/__init__.py:1762: in callHandlers\n    hdlr.handle(record)\n/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/logging/__init__.py:1028: in handle\n    self.emit(record)\n/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/site-packages/_pytest/logging.py:386: in emit\n    super().emit(record)\n/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/logging/__init__.py:1168: in emit\n    self.handleError(record)\n/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/logging/__init__.py:1160: in emit\n    msg = self.format(record)\n/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/logging/__init__.py:999: in format\n    return fmt.format(record)\n/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/site-packages/_pytest/logging.py:139: in format\n    return super().format(record)\n/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/logging/__init__.py:703: in format\n    record.message = record.getMessage()\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <LogRecord: root, 20, /home/runner/work/pythonSelenium/pythonSelenium/tests/regression_ui/test_ui_regression.py, 154, \"An error occurred: \">\n\n    def getMessage(self):\n        \"\"\"\n        Return the message for this LogRecord.\n    \n        Return the message for this LogRecord after merging any user-supplied\n        arguments with the message.\n        \"\"\"\n        msg = str(self.msg)\n        if self.args:\n>           msg = msg % self.args\nE           TypeError: not all arguments converted during string formatting\n\n/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/logging/__init__.py:392: TypeError","steps":[{"name":"Log in to Trello account.","time":{"start":1735261557476,"stop":1735261574943,"duration":17467},"status":"passed","steps":[{"name":"Navigate to the Trello login page","time":{"start":1735261557476,"stop":1735261560802,"duration":3326},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"Enter valid credentials (email, password, and 6-digits verification code)","time":{"start":1735261560802,"stop":1735261574943,"duration":14141},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":2,"attachmentsCount":0,"hasContent":true,"attachmentStep":false},{"name":"Open a board with an existing list and create a new card","time":{"start":1735261574943,"stop":1735261585295,"duration":10352},"status":"broken","statusMessage":"TypeError: not all arguments converted during string formatting\n","statusTrace":"  File \"/home/runner/work/pythonSelenium/pythonSelenium/tests/regression_ui/test_ui_regression.py\", line 154, in test_card_creation\n    log.info(\"An error occurred: \", str(e))\n  File \"/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/logging/__init__.py\", line 2216, in info\n    root.info(msg, *args, **kwargs)\n  File \"/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/logging/__init__.py\", line 1539, in info\n    self._log(INFO, msg, args, **kwargs)\n  File \"/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/logging/__init__.py\", line 1684, in _log\n    self.handle(record)\n  File \"/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/logging/__init__.py\", line 1700, in handle\n    self.callHandlers(record)\n  File \"/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/logging/__init__.py\", line 1762, in callHandlers\n    hdlr.handle(record)\n  File \"/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/logging/__init__.py\", line 1028, in handle\n    self.emit(record)\n  File \"/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/site-packages/_pytest/logging.py\", line 386, in emit\n    super().emit(record)\n  File \"/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/logging/__init__.py\", line 1168, in emit\n    self.handleError(record)\n  File \"/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/logging/__init__.py\", line 1160, in emit\n    msg = self.format(record)\n          ^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/logging/__init__.py\", line 999, in format\n    return fmt.format(record)\n           ^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/site-packages/_pytest/logging.py\", line 139, in format\n    return super().format(record)\n           ^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/logging/__init__.py\", line 703, in format\n    record.message = record.getMessage()\n                     ^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/logging/__init__.py\", line 392, in getMessage\n    msg = msg % self.args\n          ~~~~^~~~~~~~~~~\n","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":true,"stepsCount":0,"attachmentsCount":0,"hasContent":true,"attachmentStep":false}],"attachments":[{"uid":"e577a18627385fb3","name":"log","source":"e577a18627385fb3.txt","type":"text/plain","size":59081}],"parameters":[],"shouldDisplayMessage":true,"stepsCount":4,"attachmentsCount":1,"hasContent":true,"attachmentStep":false},"afterStages":[{"name":"driver::0","time":{"start":1735261585423,"stop":1735261585497,"duration":74},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"labels":[{"name":"tag","value":"TC004"},{"name":"tag","value":"TC000"},{"name":"parentSuite","value":"tests.regression_ui"},{"name":"suite","value":"test_ui_regression"},{"name":"subSuite","value":"TestUIRegression"},{"name":"host","value":"fv-az1048-752"},{"name":"thread","value":"3656-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.regression_ui.test_ui_regression"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[],"categories":[{"name":"Test defects","matchedStatuses":[],"flaky":false}],"history":{"statistic":{"failed":0,"broken":7,"skipped":0,"passed":45,"unknown":0,"total":52},"items":[{"uid":"26619be689d13440","status":"passed","time":{"start":1735237164986,"stop":1735237197666,"duration":32680}},{"uid":"242a19bacf8390f8","status":"passed","time":{"start":1735175040181,"stop":1735175072799,"duration":32618}},{"uid":"466e2c82bcd11633","status":"passed","time":{"start":1735150813440,"stop":1735150845875,"duration":32435}},{"uid":"6c1ddfc8d5d2a425","status":"passed","time":{"start":1735088685441,"stop":1735088718024,"duration":32583}},{"uid":"4b4aa7a057ea6d47","status":"passed","time":{"start":1735064356061,"stop":1735064388506,"duration":32445}},{"uid":"a4fa6e9d79da2e09","status":"passed","time":{"start":1735002279260,"stop":1735002311806,"duration":32546}},{"uid":"ee2483ddced63c84","status":"passed","time":{"start":1734977963609,"stop":1734977996107,"duration":32498}},{"uid":"8ca8d3714f3108c8","status":"passed","time":{"start":1734915961159,"stop":1734915993641,"duration":32482}},{"uid":"1d34ab60ef14fec4","status":"passed","time":{"start":1734891480902,"stop":1734891513588,"duration":32686}},{"uid":"f7ce4e0b15927cb0","status":"passed","time":{"start":1734829933207,"stop":1734829965745,"duration":32538}},{"uid":"55e7d519047b29a3","status":"passed","time":{"start":1734805086183,"stop":1734805118707,"duration":32524}},{"uid":"60a597dca206050b","status":"passed","time":{"start":1734743024242,"stop":1734743056770,"duration":32528}},{"uid":"2cb507c93d38f3d2","status":"passed","time":{"start":1734718774947,"stop":1734718807457,"duration":32510}},{"uid":"4adeaff1652acd6","status":"passed","time":{"start":1734656762073,"stop":1734656794653,"duration":32580}},{"uid":"fae969ed34d2a551","status":"passed","time":{"start":1734632404133,"stop":1734632436731,"duration":32598}},{"uid":"f5153ba1360fed18","status":"passed","time":{"start":1734570485008,"stop":1734570517629,"duration":32621}},{"uid":"9054614631c1fa13","status":"passed","time":{"start":1734546073528,"stop":1734546106204,"duration":32676}},{"uid":"7c471b428d956a38","status":"passed","time":{"start":1734484041833,"stop":1734484074539,"duration":32706}},{"uid":"1b5bca01bc230254","status":"passed","time":{"start":1734459716679,"stop":1734459749447,"duration":32768}},{"uid":"d471824c40d8acfb","status":"passed","time":{"start":1734397819757,"stop":1734397852678,"duration":32921}}]},"tags":["TC004","TC000"]},"source":"4fa5d56184fe8ea8.json","parameterValues":[]}